arch/riscv/Kconfig

```diff
@@ -225,6 +225,14 @@ config RISCV_RESERVED_IRQ_ISR_TABLES_OFFSET
 	  properly sized to take into account this offset. This is a hidden
 	  option which needs to be set per architecture and left alone.
 
+config RISCV_EMPTY_IRQ_SPURIOUS
+	bool "Create empty spurious interrupt handler"
+	default y if NRFE
```

Comment:
This should be enabled by a config of specific application, not default for nrfe.

Reply:
What if CONFIG_NRFE selected `RISCV_EMPTY_IRQ_SPURIOUS`? Would it be ok? Unless those symbols should not be connected.

Reply:
Those should not be connected - `CONFIG_NRFE` is also enabled on APP side, which does not need this optimization.

Reply:
Right, I keep confusing one core with another. Fixed.